docker/mysql/init/birthDate_create.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE birthDate (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    year TEXT,
+    count BIGINT(20),
+    timestamp TIMESTAMP NOT NULL
+);
```

Comment:
**timestamp** seems to be a not best name for column, because it is not clear to understand what this column really is.
And please, Add single blank line to the end.